src/main/java/org/broadinstitute/hellbender/Main.java

```diff
@@ -195,7 +195,9 @@ protected void handleUserException(Exception e) {
         if(printStackTraceOnUserExceptions()) {
             e.printStackTrace();
         } else {
-            System.err.println("Use -D" + STACK_TRACE_ON_USER_EXCEPTION_PROPERTY + "to print the stack trace.");
+            System.err.println("Use the system property "
+                            + STACK_TRACE_ON_USER_EXCEPTION_PROPERTY
+                            + " (--javaOptions '-DGATK_STACKTRACE_ON_USER_EXCEPTION=true') to print the stack trace.");
```

Comment:
If the gatk-launch is mentioned, it works for avoiding confusion while runing with the `java -jar` syntax. Although I think that I will override anyway as you suggested, I would like to ask/point out the following:

* Is it really necessary to point out that the system properties for gatk-launch should be set  by `--javaOptions`? I think that if a user is advance enough to print the stack-trace, they will be familiar on how to provide to gatk-launch the property. But most of the times I am wrong, so maybe it is really needed.
* One of my worries about this line hardcoded here is not only for downstream projects but for the GATK toolkit itself: what will happen if the gatk-launch option or scrip name changes? Although it is unlikely for mature problems, this can be easily overlooked and can get out of sync.